src/index.html

```diff
@@ -17,11 +17,423 @@
     />
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="./styles/styles.scss"
     />
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="LogoMoyo"
+          class="logo_img"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a
+              href="#"
+              class="is-active"
+            >
+              APPLE
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              href="#"
+              class="nav_link"
+            >
+              SAMSUNG
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              href="#"
+              class="nav_link"
+            >
+              SMARTPHONES
+            </a>
```

Comment:
rewrite header following the BEM

Reply:
I don't understand what exactly I need to change